.gitignore

```diff
@@ -0,0 +1 @@
+This file should be ignored by git
```

Comment:
As required for this project, kindly create and ignore a `test.md `file. This will help you practice ignoring files that you wouldn't want to commit to GitHub which you'll find handy in the near future.

I can see that you have tried to [set up linters](https://github.com/microverseinc/linters-config/tree/master/html-css) on the local environment, however, you did not run the npm commands required. You may wish to do this. Remember that it'll will require you to ignore the `node_modules` folder in order not to have the packages uploaded to github.

![image](https://user-images.githubusercontent.com/20554583/204671597-8f576553-0e09-468d-9194-162d12ae28ec.png)

![image](https://user-images.githubusercontent.com/20554583/204671605-e34b4693-8c15-43d1-91de-644c617c2f63.png)

Reply:
@OyePriscilla @emmiiorji Thanks for the review. Is it necessary for me to add a read me to my git flow file